java/core/src/main/java/co/worklytics/psoxy/gateway/impl/oauth/ClientCredentialsWithJWTAssertionGrantTokenRequestBuilder.java

```diff
@@ -103,7 +104,7 @@ public HttpContent buildPayload() {
 
         Map<String, String> data = new TreeMap<>();
         //https://datatracker.ietf.org/doc/html/rfc6749#section-4.4.2
-        data.put(PARAM_GRANT_TYPE, getGrantType());
+        data.put(PARAM_GRANT_TYPE, "client_credentials");
```

Comment:
A bit dirty, at some point we need make a difference from specs for this. *grantType* will appear in specs to use this builder.